### DISCUSSION TITLE
does keda support multiple external metrics from one external scaler

### DISCUSSION THREAD

**Discussion by bretagne-peiqi** [kedacore/keda#2492] [Q&A / Need Help]
we are plan to scale our app depending on http, mq, rpc multi metrics. 

does scaledobject, hpa support for multi external type metrics ?

**Comment by tomkerkhove** [comment]
You can use multiple triggers in a scaled object. Once one of them meets the scaling criteria, KEDA will start scaling.

See https://keda.sh/docs/latest/faq/
